### DISCUSSION TITLE
Get user country

### DISCUSSION THREAD

**Discussion by szanuje** [rudderlabs/rudder-sdk-js#722] [Q&A]
Hey, I configured RudderStack in my app and I see various properties being automatically passed to my Amplitude destination, including user's country. Is there a way to fetch that property within my App code?
I tried to log traits: 
```
console.log(window?.rudderanalytics?.getUserTraits());
```
but it just logs: `isLoggedIn: false`

**Comment by MoumitaM** [comment]
Hello @szanuje,
Thank you for bringing this up. 
The `rudderanalytics.getUserTraits()` method is used to fetch the traits passed in the identify call previously. This will not have any information tracked by Amplitude.

Hoping this information will help you.
